### PR TITLE
Fix context menu event listener when switching views

### LINKED ISSUE #230: Switching folder views breaks the right click context menu
The context menu adds an eventListener to on a DOM element with id `folder-view` and if this ever get's destroyed the event handler no longer works.

One example is switching the folder view.

### PR DESCRIPTION
**Pull request by MahmoudHamdy02**
Fixes #230

### CODE REVIEW

**Review by jelly — COMMENTED** [review]
This change feels like it works around the problem of the parent-id component getting destroyed when switching. My idea was to move the `parent-id` to a parent component of the CardBody/ListingTable.

**Comment by MahmoudHamdy02** [comment]
That sounds better, I've set the parent id to the `Card` component

File: src/navigator-card-body.jsx [hunk 1]
```diff
@@ -391,7 +391,7 @@ export const NavigatorCardBody = ({
         );
 
     const contextMenu = (
-        <ContextMenu parentId="folder-view">
+        <ContextMenu parentId="navigator-card">
```
Comment (jelly) [inline src/navigator-card-body.jsx:394]: This fix is alright, but this really advocates for moving the whole `<Card` into a new component class. But that's follow up territory.

**Review by jelly — APPROVED** [review]

**Comment by jelly** [comment]
Sorry, I already fixed this via https://github.com/cockpit-project/cockpit-navigator/commit/b47622846194724c79f9c33fda019e042d1fe565

**Pull request closed without merging**